test/spectests/smoketest-failures/failures.wast

```diff
@@ -0,0 +1,24 @@
+;; invalid
+(module (func (drop)))
+
+;; unlinkable
+(module (import "Mod-unnknown" "foo" (func)))
```

Comment:
Is this intentionally using `unnknown` and not `unknown` ?

Reply:
No, thanks :)